internal/pkg/build/files/files.go

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+package files
+
+import (
+	"bytes"
+	"fmt"
+	"os/exec"
+	"path/filepath"
+	"strings"
+)
+
+const script = `for n in %[1]s ; do
```

Comment:
Maybe you should use a more proper name for this?... Like:

```suggestion
const printFilesScript = `for n in %[1]s ; do
```
Or:
```suggestion
const filesScript = `for n in %[1]s ; do
```

Or whatever the script is used for...